openshift/ci-operator/generate-ci-config.sh

```diff
@@ -52,6 +52,8 @@ images:
       paths:
       - destination_dir: .
         source_path: /go/src/github.com/knative/client/kn-linux-amd64
+      - destination_dir: .
+        source_path: /go/src/github.com/knative/client/kn-linux-s390x
```

Comment:
We'll need to bring ppc64le and arm builds as well (@yselkowitz have the PR opened for v0.17.3 branch already)

Reply:
I see :sweat_smile:  well I should have checked that first. For some reason I assumed it's dealing with RPMs only. So I'll just deal with the CI template then to avoid stepping on each other toes.